users/migrations/0016_clear_sessions.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 5.0.8 on 2024-08-26 22:44
+import os
+
+from django.contrib.sessions.backends.cache import KEY_PREFIX
+from django.core.cache import cache
+from django.db import migrations
+
+
+def clear_sessions(apps, schema_editor):
+    if os.getenv("DJANGO_SETTINGS_ENVIRONMENT") not in ["AutomatedTests", "CI"]:
```

Comment:
Should not be skipped in the CI Environment since migrations are run in Platta using that environment.

Reply:
Fixed.